README_JP5.md

```diff
@@ -118,26 +118,53 @@ scp ./images/5.0.2/drivers/media/v4l2-core/videobuf-vmalloc.ko nvidia@10.0.0.116
 
 Copy them to the right places on Jetson target:
 ```
-sudo cp Image /boot
-sudo cp tegra194-p2888-0001-p2822-0000.dtb /boot/dtb
-sudo cp d4xx.ko /lib/modules/$(uname -r)/kernel/drivers/media/i2c/
-sudo cp uvcvideo.ko /lib/modules/$(uname -r)/kernel/drivers/media/usb/uvc/
-sudo cp videobuf-core.ko /lib/modules/$(uname -r)/kernel/drivers/media/v4l2-core/
-sudo cp videobuf-vmalloc.ko /lib/modules/$(uname -r)/kernel/drivers/media/v4l2-core/
+sudo cp Image /boot/d457/
+sudo cp tegra194-p2888-0001-p2822-0000.dtb /boot/d457/
+sudo cp d4xx.ko /lib/modules/$(uname -r)/updates/
+sudo cp uvcvideo.ko /lib/modules/$(uname -r)/updates/
+sudo cp videobuf-core.ko /lib/modules/$(uname -r)/updates/
+sudo cp videobuf-vmalloc.ko /lib/modules/$(uname -r)/updates/
 sudo depmod
 ```
 
 2. Edit `/boot/extlinux/extlinux.conf` primary boot option's LINUX/FDT lines to use built kernel image and dtb file:
 
     ```
-    LINUX /boot/Image-5.10.104-tegra
-    FDT /usr/lib/linux-image-5.10.104-tegra/tegra194-p2888-0001-p2822-0000.dtb
+    LINUX /boot/d457/Image
+    FDT /boot/d457/tegra194-p2888-0001-p2822-0000.dtb
     ```
 
+```
```

Comment:
Please add a sentence as" The whole file should look similar to:", before the following cat command.

Reply:
done below